lms/djangoapps/mobile_api/utils.py

```diff
@@ -1,47 +1,7 @@
 """
-Common utility methods and decorators for Mobile APIs.
+Common utility methods for Mobile APIs.
 """
-import functools
-from rest_framework import status
-from rest_framework.response import Response
-
-from lms.djangoapps.courseware.courses import get_course_with_access
-from lms.djangoapps.courseware.courseware_access_exception import CoursewareAccessException
-from opaque_keys.edx.keys import CourseKey
 from openedx.core.lib.api.view_utils import view_auth_classes
-from xmodule.modulestore.django import modulestore
-
-
-def mobile_course_access(depth=0):
```

Comment:
Why did this need to be moved out into a separate decorators file?  We should keep the 2 decorators together (see `mobile_view` below).